reference_system_autoware/src/ros2/executor/autoware_default_singlethreaded.cpp

```diff
@@ -16,9 +16,9 @@
 
 #include "reference_system_autoware/reference_system.hpp"
 
-int main(int argc, char * argv[])
-{
-  create_and_start_reference_system<rclcpp::executors::SingleThreadedExecutor>(argc, argv);
+int main(int argc, char* argv[]) {
+  create_and_start_reference_system<rclcpp::executors::SingleThreadedExecutor>(
```

Comment:
should we follow the same formatting as the `staticsinglethreaded.cpp` file?

```suggestion
  create_and_start_reference_system<
      rclcpp::executors::SingleThreadedExecutor>(argc, argv);
```